docs/lib/datastore/data-access.md

```diff
@@ -3,6 +3,14 @@ title: Manipulating data
 description: Learn how to save, query, paginate, update, delete and observe data in DataStore.
 ---
 
+## Getting started
+
+To get started, first import the `DataStore` API:
+
+```js
+import { DataStore } from '@aws-amplify/datastore';
+```
+
```

Comment:
this needs to go into a JS fragment, otherwise it will show for iOS and Android as well.

The Getting Started page mentions the imports I believe, is it something we could potentially standardize?

Reply:
Thanks, updates pushed based on this feedback.